package/MDAnalysis/units.py

```diff
@@ -301,23 +301,23 @@ def __getitem__(self, key):
     "\u212b/ps": 1.0,
     "Angstrom/picosecond": 1.0,
     "angstrom/picosecond": 1.0,  # 1
-    "Angstrom/fs": 1.0 * 1e3,
-    "Angstrom/femtosecond": 1.0 * 1e3,
-    "angstrom/femtosecond": 1.0 * 1e3,
-    "angstrom/fs": 1.0 * 1e3,
-    "A/fs": 1.0 * 1e3,
-    "Angstrom/ms": 1.0 * 1e-9,
-    "Angstrom/millisecond": 1.0 * 1e-9,
-    "angstrom/millisecond": 1.0 * 1e-9,
-    "angstrom/ms": 1.0 * 1e-9,
-    "A/ms": 1.0 * 1e-9,
-    "Angstrom/us": 1.0 * 1e-6,
-    "angstrom/us": 1.0 * 1e-6,
-    "A/us": 1.0 * 1e-6,
-    "Angstrom/microsecond": 1.0 * 1e-6,
-    "angstrom/microsecond": 1.0 * 1e-6,
-    "Angstrom/\u03BCs": 1.0 * 1e-6,
-    "angstrom/\u03BCs": 1.0 * 1e-6,
+    "Angstrom/fs": 1.0 * 1e-3,
+    "Angstrom/femtosecond": 1.0 * 1e-3,
+    "angstrom/femtosecond": 1.0 * 1e-3,
+    "angstrom/fs": 1.0 * 1e-3,
+    "A/fs": 1.0 * 1e-3,
```

Comment:
Could you please also add Å/ns (1.0 * 1e3) – for whatever reason we never added that one.

Reply:
And add an entry to CHANGELOG under Enhancements 
```
* add conversion factor for speed unit A/ns (PR #5053)
```